build_deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="rhsm"
```

Comment:
I believe this should be `rhsm-api-proxy` - it looks like we may have missed updating that in sub inv once we figured out we were in the wrong application folder